src/viam/app/data_client.py

```diff
@@ -255,10 +255,14 @@ async def tabular_data_by_sql(self, organization_id: str, sql_query: str) -> Lis
 
     async def tabular_data_by_mql(self, organization_id: str, mql_binary: List[bytes]) -> List[Dict[str, ValueTypes]]:
         """Obtain unified tabular data and metadata, queried with MQL.
+        You must install the `pymongo` package and `import bson` to use the `tabular_data_by_mql` method.
```

Comment:
Is this accurate? I don't think this is strictly necessary -- you can use any bson serialization library. I would change this to be more of a suggestion, something like:

> `mql_binary` is an array of encoded `bson` queries. You can encode your `bson` queries using a library like `pymongo`.

Reply:
well you cannot use the regular bson library that I first had installed. https://github.com/hyperopt/hyperopt/issues/547

Reply:
What happens when you try to use the `bson` library? I was just able to do this:

```python3
import bson

bson.dumps({ '$match': { 'location_id': '<location-id>' }})
```

And that outputs a valid bson binary. I'm curious as to the functionality you are seeing.

Reply:
that does indeed work. You can't call encode on that. I think that's what I was trying before and it has just a varietey of `encode_*` methods